CHECKLIST.md

```
@@ -107,6 +107,24 @@ WGPU `90048506717`, and Metal `90048504635` passed; Coeus consumer jobs CUDA
 passed. Hardware-device jobs skipped because no registered runner was
 available. ADR 0028 owns the exact GELU contract.
 
+## ATLAS-COEUS-HEPHAESTUS-ACTIVATION-TAIL-PARITY-001 [arch][minor]
```

markdownlint-cli2 (0.23.0)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "minor"

(MD052, reference-links-images)